### ISSUE TITLE
tasks/uniquify_people searches are bent or broken

### ISSUE THREAD

**Issue by jrflood** [SpeciesFileGroup/taxonworks#622]
Previously working inter-task link to tasks/uniquify_people/index?last_name=<last name> no longer enables the find person button.  Apparently this section was modified to require both names (which could be added to the request from the referring page).

Adding a first name and clicking [Find Person] yields 25 people unrelated in any way to the name, but populates the autocomplete search box with the last name of the left-hand search (and appears to completely ignore it when filled from [Find Person]).

**Comment by mjy** [comment]
Couple points
* Only last name should be required for search
* First name only searches should likely not be allowed
* The similar people is a single function, abstracted out now, so the functionality can be easily modified. Rather than use wildcards I changed it to experiment with using Levenshtein distance.  It can be modified if we want, but I'd like real users(TM) to provide feedback before we change this.

**Comment by jrflood** [comment]
RE: your first two points: I concur generally.  But, since the single param last_name now does not work as originally designed (does not enable the [Find Person] button, I added the first letter of the first name to the passed-in last name in order to enable the find button (not a first name only search).  This different behavior is apparently due to recent modifications to the Vue app after the original pull request.

The subsequent reaction to [Find Person] in the Vue app returns  a seemingly completely unrelated list of 25 people ordered by last name -- but in fact has a match to the single letter "M" in the first name after a longer look.

It appears I selected my favorite person and induced a behavior that looked MUCH worse than it really is by only supplying his first name initial to proceed through the form.  It may work right if the logic for the search button is revised, since the autocomplete seems to work.  I'll see if it's obvious how the logic is changed.
